src/control/Control.cpp

```diff
@@ -1246,7 +1246,7 @@ void Control::insertPage(const PageRef& page, size_t position) {
     int visibleHeight = 0;
     scrollHandler->isPageVisible(position, &visibleHeight);
 
-    if (visibleHeight < 10) {
+    if (visibleHeight < 100) {
```

Comment:
What does this constant do?

Reply:
When 100pt are visible don't scroll. Its the height of the visible area of the page, not zoomed. I am not sure why we have that behaviour.

Reply:
I assume this prevents scrolling while pages are not yet rendered correctly. But considering that we actually know the size of the page from the start this does not really make sense. Would be interesting when this was introduced. Might give a hint why it is required.